browse/controllers/abs_page/__init__.py

```diff
@@ -77,24 +78,31 @@ def get_abs_page(arxiv_id: str,
         response_data['author_links'] = \
             split_long_author_list(queries_for_authors(abs_meta.authors), truncate_author_list_size)
         response_data['author_search_url_fn'] = search_author
-        response_data['include_inspire_link'] = include_inspire_link(abs_meta)
-        response_data['dblp'] = _check_dblp(abs_meta)
-        response_data['trackback_ping_count'] = count_trackback_pings(arxiv_id)
 
         # Dissemination formats for download links
         add_sciencewise_ping = _check_sciencewise_ping(abs_meta.arxiv_id_v)
         response_data['formats'] = metadata.get_dissemination_formats(
-                                    abs_meta,
-                                    download_format_pref,
-                                    add_sciencewise_ping)
-        # Ancillary files
-        response_data['ancillary_files'] = \
-            metadata.get_ancillary_files(abs_meta)
-
-        # Browse context
-        _check_context(arxiv_identifier,
-                       request_params,
-                       response_data)
+            abs_meta,
+            download_format_pref,
+            add_sciencewise_ping)
+
+        # the following are less critical and the template must display without them
+        try:
+            response_data['include_inspire_link'] = include_inspire_link(abs_meta)
+            response_data['dblp'] = _check_dblp(abs_meta)
+            response_data['trackback_ping_count'] = count_trackback_pings(arxiv_id)
+
+            # Ancillary files
+            response_data['ancillary_files'] = \
+                metadata.get_ancillary_files(abs_meta)
+
+            # Browse context
+            _check_context(arxiv_identifier,
+                           request_params,
+                           response_data)
+        except Exception as e:
+            app.logger.error("Error getting non-critical abs page data", exc_info=app.debug)
```

Comment:
Could we use logging from arxiv-base for consistency with the other repos? e.g.
```
from arxiv.base import logging
logger = logging.getLogger(__name__)
...
logger.error(f'Error getting non-critical abs page data: {e}')
```
See also https://github.com/cul-it/arxiv-base/blob/master/arxiv/base/logging.py

Reply:
I think that base is going to set the logger on app during the call to Base(app) and that this is all we need to do. I might be wrong, I'll check.

Reply:
I've changed this to use the arxiv.base logging pattern above.